doc_src/intro.rst

```diff
@@ -0,0 +1,19 @@
+This repo is very much focused on the instrumentation aspect of our lab (e.g., Instek oscilloscope, Instek frequency generator, HP microwave Frequency source, Bridge12, Gigatronics power meter). A primary server 'FLInst' acts as the glue between all instruments for user friendly integration. 
+
+FLInst
+======
+This package includes a power control server that acts as an integral communication between instrumentation as well as between the computer running SpinCore and the computer running XEPR for seamless acquisition. The FLInst allows the user to set desired fields, initiate communication between instruments, as well as initiating several GUI programs for streamlining the acquisition process.
+
```

Comment:
general description of FLInst -- I don't think I'm using the proper phrase with calling FLInst a server... what would be the noun I call it?